system-addon/lib/TelemetryFeed.jsm

```diff
@@ -274,6 +278,32 @@ this.TelemetryFeed = class TelemetryFeed {
   }
 
   /**
+   * handlePagePrerendered - Set the session as prerendered
+   *
+   * @param  {string} portID the portID of the target session
+   */
+  handlePagePrerendered(portID) {
+    const session = this.sessions.get(portID);
+
+    if (!session) {
+      // It's possible the tab was never visible – in which case, there was no user session.
+      return;
+    }
+
+    session.perf.is_prerendered = true;
+  }
+
+  /**
+   * handleNewTabInit - Handle NEW_TAB_INIT
```

Comment:
This wants a bit more commentary describing what it does.  It also wants a test to check that it calls addSession.